### DISCUSSION TITLE
已经19小时订阅无更新了，是有BUG吗

### DISCUSSION THREAD

**Discussion by geminidesdd** [gkd-kit/gkd#91] [General]
![Screenshot_20231002_090348.jpg](https://github.com/gkd-kit/gkd/assets/146546137/ce8a7ba1-27ca-4be5-b4ea-8f1150cbd97d)

**Comment by lisonge** [comment]
> 是有BUG吗,

不是

---

<https://github.com/gkd-kit/subscription>

![image](https://github.com/gkd-kit/gkd/assets/38517192/53dc0a52-9873-4358-8585-8fe2f4947712)

订阅不更新是因为版本号没有更新
